modules/standard/IO.chpl

```diff
@@ -6972,23 +6973,110 @@ proc fileWriter.styleElement(element:int):int {
   Iterate over all of the lines ending in ``\n`` in a :record:`fileReader` - the
   fileReader lock will be held while iterating over the lines.
 
-  Only serial iteration is supported. This iterator will halt on internal
-  system errors.
+  This iterator will halt on internal :ref:`system errors<io-general-sys-error>`.
```

Comment:
Right but we'd like to change that in the future, presumably, once we have smoothed over throwing iterators? Could you add a note about that here and a Future Work note in the PR message?